examples/vite-demo-vanilla-bundle/src/examples/example11.ts

```diff
@@ -175,7 +175,7 @@ export default class Example11 {
         type: FieldType.date, outputType: FieldType.dateIso,
         filterable: true,
         filter: { model: Filters.compoundDate },
-        editor: { model: Editors.date, massUpdate: true },
+        editor: { model: Editors.date, massUpdate: true, disabled: false, editorOptions: { allowInput: true } },
```

Comment:
is there a particular reason to add the `disabled: false`?

Reply:
just an oversite from a manual test. good catch